### DISCUSSION TITLE
load whisper local model

### DISCUSSION THREAD

**Discussion by silvacarl2** [openai/whisper#1433] [Q&A]
how can we load whisper fine tuned local model for transcription?

it keeps trying to load a model from HF, we want to make it use our fine tuned model for testing and we want it to load from the local checkpoint.

carl

**Comment by phineas-pta** [comment]
https://github.com/openai/whisper/discussions/830#discussioncomment-4652413

**Comment by silvacarl2** [comment]
does this mean we have to upload to HF to test a fine tuned model locally?

**Reply by phineas-pta** [reply]
no, did you read the code in comment ?

**Reply by silvacarl2** [reply]
i think i got it now, trying that out!

**Reply by silvacarl2** [reply]
ok, totally works!
